exercises/kindergarten-garden/example.py

```diff
@@ -1,18 +1,31 @@
 class Garden(object):
 
-    __plant_names = {"C": "Clover", "G": "Grass",
-                     "R": "Radishes", "V": "Violets"}
+    STUDENTS = [
```

Comment:
Since this is already a separate file/module, is it redundant to nest `STUDENTS` and `PLANTS` under `Garden`?

Reply:
Judgement call ... STUDENTS could be seen as independent of the **Garden** class, but **PLANTS** is pretty bound to the class. I've seen a number of students prefer to have the lookup table as a class constant, and by making them both class bound the file's a bit cleaner to read.